src/components/Person/Person.jsx

```diff
@@ -1 +1,35 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => {
+  const {
+    name,
+    age,
+    sex,
+    partnerName,
+    isMarried,
+  } = person;
+
+  const checkSex = sex === 'f'
+    ? 'husband'
+    : 'wife';
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        {`My name is ${name}`}
+      </h2>
+
+      {age > 0 && (
+        <p className="Person__age">
+          {`I am ${age}`}
+        </p>
+      )}
```

Comment:
cool!

also u can use [Nullish coalescing operator (??)](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing)

```suggestion
      {age ?? (
        <p className="Person__age">
          {`I am ${age}`}
        </p>
      )}
```